Extensions/Screenshot/screenshottools.js

```diff
@@ -0,0 +1,37 @@
+/**
+ * @memberof gdjs
+ * @class screenshot
+ * @static
+ * @private
+ */
+
+gdjs.screenshot = {}
+
+/**
+ * Save a screenshot of the game.
+ * @param {string} savepath The path where to save the screenshot
+ */
+gdjs.screenshot.takeScreenshot = function (runtimeScene, savePath) {
+  //const electron = gdjs.RuntimeGamePixiRenderer.getElectron();
+  const electron = runtimeScene.getGame().getRenderer().getElectron();
+  let canvas = null;
+  let content = null;
+  let fileSystem = null;
```

Comment:
You don't need to declare variables at the top like this, you can declare them as you use them.
Bonus point: use const to avoid accidental reuse of the variable:

```
const fs = electron.remote.require("fs");
const canvas = runtimeScene.getGame().getRenderer().getCanvas();
...
```